markovs_household/data/appliance.py

```diff
@@ -61,4 +73,50 @@ class Appliance:
     """
 
     appliance_type: ApplianceType
-    operation_intervals: List[TimeInterval]
+    _operation_intervals: List[TimeInterval] = field(default_factory=list)
+    random_generator: ClassVar[random.Random] = random.Random(42)
+
+    @property
+    def __operation_intervals(self):
+        return self._operation_intervals

```

Comment:
In my understanding, variables and methods starting with two underscores denote private variables/methods. Can you enlighten me as to what purpose this getter then has?